stories/atoms/components/toast/Toast.stories.mdx

```diff
@@ -316,6 +321,62 @@ These two elements are optional and are not inter-related.
               severityLevel="success"
             />
           </div>
+          <div>
+            <Button
+              label="Bot Anik"
```

Comment:
Please, add a better label for the toast button that will be more explicit for the user

Reply:
Good point! 👍